pkg/controller/job/controller.go

```diff
@@ -110,38 +116,39 @@ func NewJobController(kubeClient clientset.Interface, resyncPeriod controller.Re
 		},
 	)
 
-	jm.podStore.Store, jm.podController = framework.NewInformer(
-		&cache.ListWatch{
-			ListFunc: func(options api.ListOptions) (runtime.Object, error) {
-				return jm.kubeClient.Core().Pods(api.NamespaceAll).List(options)
-			},
-			WatchFunc: func(options api.ListOptions) (watch.Interface, error) {
-				return jm.kubeClient.Core().Pods(api.NamespaceAll).Watch(options)
-			},
-		},
-		&api.Pod{},
-		resyncPeriod(),
-		framework.ResourceEventHandlerFuncs{
-			AddFunc:    jm.addPod,
-			UpdateFunc: jm.updatePod,
-			DeleteFunc: jm.deletePod,
-		},
-	)
+	podInformer.AddEventHandler(framework.ResourceEventHandlerFuncs{
+		AddFunc:    jm.addPod,
+		UpdateFunc: jm.updatePod,
+		DeleteFunc: jm.deletePod,
+	})
+	jm.podStore.Store = podInformer.GetStore()
```

Comment:
> jm.podController = podInformer.GetController()
> 
> is missing here.

There is no such field in this controller.  Turns out it we don't need it for this one.